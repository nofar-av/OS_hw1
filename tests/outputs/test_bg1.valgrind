==26284== Memcheck, a memory error detector
==26284== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==26284== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==26284== Command: /home/omer/OS_wet1/smash
==26284== Parent PID: 26282
==26284== 
==26284== 
==26284== FILE DESCRIPTORS: 4 open at exit.
==26284== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_bg1.valgrind
==26284==    <inherited from parent>
==26284== 
==26284== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_bg1.err
==26284==    <inherited from parent>
==26284== 
==26284== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_bg1.out
==26284==    <inherited from parent>
==26284== 
==26284== Open file descriptor 0:
==26284==    <inherited from parent>
==26284== 
==26284== 
==26284== HEAP SUMMARY:
==26284==     in use at exit: 0 bytes in 0 blocks
==26284==   total heap usage: 29 allocs, 29 frees, 82,288 bytes allocated
==26284== 
==26284== All heap blocks were freed -- no leaks are possible
==26284== 
==26284== For counts of detected and suppressed errors, rerun with: -v
==26284== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
