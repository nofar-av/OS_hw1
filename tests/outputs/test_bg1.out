smash> smash: got ctrl-Z
smash: process 28230 was stopped
smash> [1] sleep 100 : 28230 0 secs (stopped)
smash> sleep 100 : 28230
smash> [1] sleep 100 : 28230 0 secs
smash> smash: sending SIGKILL signal to 1 jobs:
28230: sleep 100
